src/components/ComponentGallery/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./index.css";
+// eslint-disable-next-line import/no-unresolved
+import pagesData from "../../icds-pages-data.json";
+
+interface ComponentDetails {
+  title: string;
+  subTitle: string;
+  path: string;
+}
+
+const ComponentGallery: React.FC = () => {
+  const componentDetails: ComponentDetails[] = [];
+
+  pagesData.data.forEach((page) => {
+    if (page.tabs) {
+      componentDetails.push({
+        title: page.title,
+        subTitle: page.subTitle,
+        path: page.path,
+      });
+    }
+  });
+
+  const uniqueComponentDetails = [
+    ...new Map(componentDetails.map((item) => [item.title, item])).values(),
+  ];
+
+  return (
+    <ul className="card-container">
+      {uniqueComponentDetails &&
+        uniqueComponentDetails.map((item: ComponentDetails) => (
+          <li>
+            <ic-card
```

Comment:
```suggestion
            <GatsbyLink to={`${item.path}`}>
              <ic-card
                heading={item.title}
                message={item.subTitle}
                full-width
                clickable
                />
            </GatsbyLink>
```

Reply:
Links don't work so might need to instead use GatbsyLink to fix that

Reply:
Once deployed GatbsyLink is needed - thanks :)

Reply:
Have updated with GatbsyLink and the deployment link is not working either. However both ways work on the local.